opentech/apply/funds/templates/funds/base_submissions_table.html

```diff
@@ -43,6 +43,7 @@
 {% block extra_js %}
     {{ filter.form.media.js }}
     <script src="{% static 'js/apply/all-submissions-table.js' %}"></script>
+    <script src="https://cdn.jsdelivr.net/npm/symbol-es6@0.1.2/symbol-es6.min.js"></script>
```

Comment:
Sorry, one other point I missed: what is this library used for? Doesn't appear to be anything relating to it in the rest of the MR

Reply:
the polyfill below this one urlSearchParams requires it for IE, unsure why it's not included in it already